pkg/core/database/heavy/transactions.go

```diff
@@ -194,7 +194,18 @@ func (t transaction) StoreBlock(b *block.Block) error {
 	value = b.Header.Hash
 	t.put(key, value)
 
-	return nil
+	// Delete expired bid values
+	key = BidValuesPrefix
+	iterator := t.snapshot.NewIterator(util.BytesPrefix(key), nil)
+
+	for iterator.Next() {
+		height := binary.LittleEndian.Uint64(iterator.Key()[1:])
```

Comment:
We don't expect partial writes but just in case we can check the len if it's 1+8 here too.